spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -32,36 +32,36 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
-[[microprofile4.0-RC1]]
-=== MicroProfile 4.0 Release Candidate 1 (Q42020)
-// MicroProfile 4.0-RC1 is the 14th platform release for the
+[[microprofile4.0-RC2]]
+=== MicroProfile 4.0 Release Candidate 2 (Q12021)
```

Comment:
I think we can get RC2 (and final) released in 2020 based on [Wayne's email](https://www.eclipse.org/lists/microprofile-wg/msg00209.html). We should bring this up in the microprofile-wg alias. There is nothing holding us back if all specs are voted to pass.